src/screens/HomeScreen.js

```diff
@@ -99,11 +99,96 @@ export const HomeScreen = ({ navigation, route }) => {
     }, 500);
   };
 
+  const flatListData = [
```

Comment:
why not simply `data` name here?

Reply:
I named it `flatListData` to show that data is the data of `FlatList`, but I'll change it.

Reply:
and if we change the flat list to another list or a scrollview somewhen, you would rename the variable here? the name should not contain something about where it is used but what it is.

Reply:
No, I wouldn't. You're right. 👍☺️